### DISCUSSION TITLE
The scene failed to load

### DISCUSSION THREAD

**Discussion by victormunoz** [mozilla/hubs#4863] [General Questions & Discussion] [answered]
I am following the quick start guide on a Winows 10 and Chrome.
After the command `npm run dev ` I navigate to [localhost](https://localhost:8080/), accept the warning, and eveything seems to work fine.
But when I try to create a room the message "The scene failed to load" appears.

There are several error messages in the browser console, the most serious provably being:
Access to XMLHttpRequest at 'https://hubs-proxy.com/https://hubs-upload-cdn.com/files/892dc759-265b-4ebc-9894-765386f0def2.bin' from origin 'https://localhost:8080' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

And then
Failed to load glTF model

PS. Seems similar to this issue: https://github.com/mozilla/hubs/issues/4813 but I can see the app logo image correctly.

Thanks!

**Comment by matthewbcool** [comment, marked as answer]
Setup hubs local hostname:
https://github.com/mozilla/reticulum#1-setup-the-hubslocal-hostname

then try on `https://hubs.local:8080/`

**Reply by victormunoz** [reply]
In Windows the file to edit is C:\Windows\System32\drivers\etc\hosts
